src/logic/delete.ts

```diff
@@ -0,0 +1,61 @@
+import i18n from "@dhis2/d2-i18n";
+
+import Instance from "../models/instance";
```

Comment:
On Instances pages I get: "Cannot read property 'modelValidations' of undefined". That's because we need to use the d2-ui-components branch `feature/objects-table-improvements-10`?

Reply:
Yeah, done!